src/content/docs/en/pages/guides/marketplace/templates/wordpress-instacreator.mdx

```diff
@@ -80,25 +81,14 @@ Fields identified with an asterisk are mandatory.
 
 After completing all the information, click the **Next** button, located in the bottom-right corner. This will start the deployment process.
```

Comment:
```suggestion
After completing all the information, click the **Save** button, located in the bottom-right corner. This will start the deployment process.
```